<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>379</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SF UI Display</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Maze</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;SF UI Display&quot;;
background-color: rgb(70, 70, 70);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QTableWidget" name="maze">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="150">
            <red>255</red>
            <green>109</green>
            <blue>25</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>255</red>
            <green>77</green>
            <blue>71</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>SF UI Display</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget 
{
    gridline-color: rgb(150,150,150);
	font: 11pt &quot;SF UI Display&quot;;
}

QTableView::item:focus
{
   selection-background-color: rgb(255, 177, 112);
}</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>35</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>35</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="set_start">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
   border: 1px solid gray;
	color : rgb(243,243,243);
	font: 12pt &quot;SF UI Display&quot;;
	background-color : rgb(255, 73, 73);
}

QPushButton:pressed 
{
	color : rgb(255, 73, 73);
	background-color : rgb(243, 243, 243);
}

QPushButton:disabled
{
background-color : rgb(60, 60, 60);
}</string>
        </property>
        <property name="text">
         <string>Set Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_finish">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
   border: 1px solid gray;
	color : rgb(243,243,243);
	font: 12pt &quot;SF UI Display&quot;;
	background-color : rgb(255, 73, 73);
}

QPushButton:pressed 
{
	color : rgb(255, 73, 73);
	background-color : rgb(243, 243, 243);
}

QPushButton:disabled
{
background-color : rgb(60, 60, 60);
}</string>
        </property>
        <property name="text">
         <string>Set Finish</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_wall">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
   border: 1px solid gray;
	color : rgb(243,243,243);
	font: 12pt &quot;SF UI Display&quot;;
	background-color : rgb(255, 73, 73);
}

QPushButton:pressed 
{
	color : rgb(255, 73, 73);
	background-color : rgb(243, 243, 243);
}

QPushButton:disabled
{
background-color : rgb(60, 60, 60);
}</string>
        </property>
        <property name="text">
         <string>Set Wall</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="but_start">
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
   border: 1px solid gray;
	color : rgb(243,243,243);
	font: 12pt &quot;SF UI Display&quot;;
	background-color : rgb(103, 236, 118);
}

QPushButton:pressed 
{
	color : rgb(103, 236, 118);
	background-color : rgb(243, 243, 243);
}

QPushButton:disabled
{
background-color : rgb(60, 60, 60);
}</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>SF UI Display</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(213,213,213);</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;SF UI Display&quot;;
color: rgb(255,255,255);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSize"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="text">
    <string>Size</string>
   </property>
   <property name="toolTip">
    <string>changesize</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
